### PR TITLE
i18n(es): translate `card-grids`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- What does this PR change? Give us a brief description.
Translates `card-grids` to spanish

### CODE REVIEW

File: docs/src/content/docs/es/components/card-grids.mdx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+import { Card, CardGrid, LinkCard } from '@astrojs/starlight/components';
+
+Para envolver múltiples componentes [`<Card>`](/es/components/cards/) o [`<LinkCard>`](/es/components/link-cards/) en una cuadrícula, usa el componente `<CardGrid>`.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<CardGrid slot="preview">
+	<Card title="Stars" icon="star">
```
Comment (HiDeoo) [inline docs/src/content/docs/es/components/card-grids.mdx:17]:
```suggestion
	<Card title="Estrellas" icon="star">
```

I guess we can use the previous translation from the old component documentation?

File: docs/src/content/docs/es/components/card-grids.mdx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+		Sirius, Vega, Betelgeuse
+	</Card>
+	<Card title="Moons" icon="moon">
```
Comment (HiDeoo) [inline docs/src/content/docs/es/components/card-grids.mdx:20]:
```suggestion
	<Card title="Lunas" icon="moon">
```

File: docs/src/content/docs/es/components/card-grids.mdx [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+{% /cardgrid %}
+```
+
+</Fragment>
+
+<CardGrid slot="preview">
+	<Card title="Check this out" icon="open-book">
+		Contenido interesante que deseas destacar.
+	</Card>
+	<Card title="Other feature" icon="information">
```
Comment (HiDeoo) [inline docs/src/content/docs/es/components/card-grids.mdx:74]: We should be able to translate both titles in this example.

File: docs/src/content/docs/es/components/card-grids.mdx [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+</Preview>
+
+### Agrupar tarjetas de enlace
+
+Muestra varios componentes [`<LinkCard>`](/es/components/link-cards/) uno al lado del otro cuando hay suficiente espacio agrupándolos con el componente `<CardGrid>`.
+
+<Preview>
+
+```mdx {3,6}
```
Comment (HiDeoo) [inline docs/src/content/docs/es/components/card-grids.mdx:87]:
```suggestion
```mdx {3,9}
```

To match the new formatting

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left a few comments and suggestions but everything else looks good to me.

**Comment by dreyfus92** [comment]
Thanks @HiDeoo 🫡

**Review by HiDeoo — APPROVED** [review]
Thanks for updating the PR :thumbsup:

**Pull request merged**